xml/System.Net.Sockets/IOControlCode.xml

```diff
@@ -697,7 +697,7 @@
       </ReturnValue>
       <MemberValue>3355443207</MemberValue>
       <Docs>
-        <summary>Retrieve the QOS structure associated with the socket. This control is only supported on platforms that provide a QOS capable transport (Windows Me, Windows 2000, and later.) This value is equal to the Winsock 2 SIO_GET_QOS constant.</summary>
+        <summary>Retrieve the QOS structure associated with the socket. This control is only supported on platforms that provide a QOS capable transport (Windows 2000, and later.) This value is equal to the Winsock 2 SIO_GET_QOS constant.</summary>
```

Comment:
"Windows 2000 and later" can be excised. Maybe we can kill the sentence "This control code is only supported..." entirely? /cc @wfurt